web/ui/templates/status.html

```diff
@@ -54,7 +54,8 @@ <h2 id="alertmanagers">Alertmanagers</h2>
         </tr>
         {{range .Alertmanagers}}
         <tr>
-          <td>{{.}}</td>
+          {{/* Alertmanager URLs always have Scheme, Host and Path set */}}
+          <td>{{.Scheme}}://<a href="http://{{.Host}}">{{.Host}}</a>{{.Path}}</td>
```

Comment:
Why is `{{ .Scheme }}` outside of the link and inside it is hardcoded to `http://`?